src/main/kotlin/no/nav/medlemskap/common/Metrics.kt

```diff
@@ -89,6 +90,11 @@ fun harIkkeArbeidsforhold12MndTilbakeCounter(): Counter = Counter
         .description("counter for brukere som ikke har arbeidsforhold som starter 12 mnd tilbake")
         .register(Metrics.globalRegistry)
 
+fun dekningKoderCounter(orEmpty: String): Counter = Counter
+        .builder("dekningstyper_koder")
```

Comment:
Burde bruke tags i dette tilfellet. Se på stillingsprosentCounter eller regelCounter for eksempel :)